### PR TITLE
Draft:  Allow binary testing in codeclimate

### PR DESCRIPTION
**Pull request by GarethCabournDavies**
There are a few things not being tested by codeclimate - is this something we can sort?

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -56,4 +56,5 @@ exclude_patterns:
 - tools/*
 - examples/*
 - test/*
+- "!bin/*"
```
Comment (spxiwh) [inline .codeclimate.yml:59]: IIRC It's already looking in bin ... It's just not looking at things that are not `.py` codes.

Comment (GarethCabournDavies) [reply .codeclimate.yml:59]: Yeah, this was the first (and simplest) test to check that

Comment (GarethCabournDavies) [reply .codeclimate.yml:59]: You're right, the documentation [says](https://docs.codeclimate.com/docs/excluding-files-and-folders).

> Furthermore, engines only run their analysis on files with an appropriate file extension for the language being analyzed (.rb for a Ruby engine, .py for a Python engine, etc). Therefore, in most cases you should not need to specify excludes for files in languages other than your target languages

I've asked stackoverflow for any help on this

**Comment by GarethCabournDavies** [comment]
This should really be an issue

**Pull request closed without merging**